com.makemkv.MakeMKV.json

```diff
@@ -0,0 +1,143 @@
+    /* Network access */
+    "--share=network",
+    /* For optical drive access */
+    "--device=all",
+    "--env=JAVA_HOME=/app/jre"
+  ],
+  "command": "makemkv",
+  "build-options" : {
+    "append-path": "/usr/lib/sdk/openjdk10/bin"
+  },
+  "rename-desktop-file": "makemkv.desktop",
+  "rename-icon": "makemkv",
+  "cleanup": [
+    "/include",
+    "/lib/debug",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "openjdk",
+        "buildsystem": "simple",
+        "build-commands": [
+          "/usr/lib/sdk/openjdk10/install.sh"
+        ]
+    },
+    {
+      "name": "x264",
+      "config-opts": [
+        "--disable-cli",
+        "--enable-shared",
+        "--disable-asm"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.videolan.org/x264/snapshots/x264-snapshot-20181110-2245-stable.tar.bz2",
+          "sha256": "44d843291719dfd340b854c51360fbed1ddcb40580b7a239eccae4ca558c3868"
+        }
+      ]
+    },
+    {
+      "name": "ffmpeg",
+      "config-opts": [
+        "--enable-gpl",
+        "--disable-static",
+        "--enable-shared",
+        "--disable-doc",
+        "--disable-ffplay",
+        "--disable-ffprobe",
+        "--enable-libopus",
+        "--enable-libvpx",
+        "--enable-libx264"
+      ],
+      "post-install": [
+        "chrpath -d /app/bin/ffmpeg"
+      ],
+      "cleanup": [
+        "/share/ffmpeg/examples"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz",
+          "sha256": "a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-oss",
+      "buildsystem": "autotools",
+      "cflags": "-D__GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -D __STDC_FORMAT_MACROS",
+      "config-opts": [
+        "--enable-allcodecs"
+      ],
+      "make-install-args": [
+        /* really just to avoid call to ldconfig */
+        "DESTDIR=/"
+      ],
+      "post-install": [
+        "install -Dp -m 644 com.makemkv.MakeMKV.appdata.xml --target-directory=/app/share/appdata"
+      ],
+      "sources" : [
+        {
+          "type": "archive",
+          "url": "http://www.makemkv.com/download/makemkv-oss-1.14.1.tar.gz",
+          "sha256": "fc8e15ef1148988b241110fc7cf2c2d916abf01bb85c3450b4e2ee10aa054b7b"
+        },
+        {
+          "type": "file",
+          "path": "com.makemkv.MakeMKV.appdata.xml"
+        }
+      ]
+    },
+    {
+      "name": "makemkv-bin",
+      "buildsystem": "simple",
+      "build-commands": [
+        /* TODO: put eula in appstream data once it supports it */
+        /* NOTE: author allows redistribution as long as eula is referenced in package description */
+        "mkdir -p tmp",
+        "echo accepted > tmp/eula_accepted",
+        "make install DESTDIR=/ PREFIX=/app"
+      ],
+      "sources" : [
+        {
```

Comment:
If it's proprietary and doesn't allow redistribution, you need to use extra-data instead.

Reply:
The auther allows redistribution and skipping the eula as long as the package description references the eula.  See:

https://www.makemkv.com/forum/viewtopic.php?f=3&t=3437&p=12765#p12765